ui/src/pages/insightDetail/components/eventAggregator/index.tsx

```diff
@@ -350,6 +351,9 @@
     ? { width: '100%', height: 300 }
```

Comment:
Consider adding a null check before accessing `contentRef.current` as it could be null if the ref hasn't been attached yet. You can use optional chaining: `contentRef.current?.getBoundingClientRect()?.top`.